resources/definitions/lulzbot_taz_pro_dual.def.json

```diff
@@ -0,0 +1,34 @@
+{
+   "version": 2,
+   "name": "Lulzbot TAZ Pro Dual Extruder",
+   "inherits": "lulzbot_taz_pro_common",
+   "metadata": {
+        "author": "Tom Callaway <spot@fedoraproject.org>",
+        "manufacturer": "Fargo Additive Manufacturing Equipment 3D, LLC",
+        "visible": true,
+        "file_formats": "text/x-gcode",
+        "platform": "lulzbot_taz_pro_bed.stl",
+        "platform_offset": [0, -25, 0],
+        "machine_extruder_trains": {
+            "0": "lulzbot_taz_pro_dual_extruder_0",
+            "1": "lulzbot_taz_pro_dual_extruder_1"
+        }
+   },
+
+   "overrides": {
+       "machine_name": { "default_value": "Lulzbot TAZ Pro" },
+       "machine_width": { "default_value": 282 },
+       "machine_depth": { "default_value": 286 },
+       "machine_height": { "default_value": 286 },
```

Comment:
In the [specifications](https://shop.lulzbot.com/lulzbot-taz-3d-printer-pro-boxed-for-retail-na-kt-pr0050na) it says 280x280x285. Are the dimensions here correct?

Reply:
Those values were taken from https://gitlab.com/lulzbot3d/cura-le/cura-lulzbot/-/blob/master/resources/definitions/lulzbot_quiver_evergreen_bagworm.def.json. I am not sure why there is a slight disconnect. My initial thought is that perhaps the dimensions are printable area, but since the printer has a wipe pad on each side, it needs to be able to move into that space.